app/src/main/java/com/getsensibill/spendmanager/demo/fragment/FragmentJavaActivity.java

```diff
@@ -70,15 +54,16 @@ public void onDisplayNetworkError(boolean networkNotAvailable) {
         bundle.putBoolean(WebUiNetworkErrorFragment.ARG_NO_NETWORK_ERROR, networkNotAvailable);
         fragment.setArguments(bundle);
 
+        String TAG_WEB_NETWORK_ERROR_FRAGMENT = "TAG_WEB_NETWORK_ERROR_FRAGMENT";
```

Comment:
this should be a class level `private static final String` no?

Reply:
made them private final. Not sure it needs static?

Reply:
made static